components/List/index.js

```diff
@@ -1,23 +1,27 @@
-import React, {Component} from 'react'
+import React from 'react'
 import {StyleSheet, View, Text} from 'react-native'
 import PropTypes from 'prop-types'
+import Card from '../Card'
 
-export default List = props => (
-    static propTypes = {
-        props.title = PropTypes.string.isRequired,
-        props.cards = PropTypes.array.isRequired,
-    }
+export default List
```

Comment:
Convention is to put this at the bottom of the file for stateless functional components